contracts/escrow/LemonadeEscrowFactoryV1.sol

```diff
@@ -10,11 +10,19 @@ import "./LemonadeEscrowV1.sol";
 
 contract LemonadeEscrowFactoryV1 is Ownable, ILemonadeEscrowFactory {
     address _signer;
+    address _feeCollector;
+    uint256 _feeAmount;
 
     event EscrowCreated(address escrow);
 
-    constructor(address _initialSigner) {
+    constructor(
+        address _initialSigner,
```

Comment:
This is still not the right convention, only private variables should be prefixed with _